base/util/src/main/java/com/netscape/cmsutil/crypto/CryptoUtil.java

```diff
@@ -2758,6 +2861,56 @@ public static PrivateKey unwrap(CryptoToken token, PublicKey pubKey, boolean tem
         return pk;
     }
 
+     public static SymmetricKey unwrapAESSKeyFromBytes(CryptoToken token, byte[] inputKeyArray,
```

Comment:
I think the name of this function might be a misnomer.  The purpose of the method is to import a sym key (in bytes) onto the token by wrapping/unwrapping.  I suggest that you
1. add a comment to explain what it does
2. maybe change the name to something like unwrapSymKeyFromBytesToToken ?  Actually, if there's a similar function for non-aes keys in jss or pki, you could maybe follow the same naming convention?

Reply:
Note the name already has an S in there. unwrapAESSKeyFromBytes. I made it AES cause the routine was specific to AES.